README.md

```diff
@@ -26,6 +27,9 @@ The [WAI-ARIA](https://www.w3.org/TR/wai-aria-1.1/) spec is deliberately not dup
 - Font weights below 400 should not be used
 - Medium sized headings generally look best with a font weight between 500-600
 - Adjust values fluidly by using CSS [`clamp()`](https://developer.mozilla.org/en-US/docs/Web/CSS/clamp), e.g. `clamp(48px, 5vw, 72px)` for the `font-size` of a heading
+- Tabular figures should be used in lists where making comparisons between numbers are important, or layout shifts are undesirable, like in timers
```

Comment:
Could you elaborate on what "making comparisons between numbers" means? I understand how this helps with not having layout shift but that part is fuzzy for me. Perhaps we can also mention `font-variant-numeric: tabular-nums`?

Reply:
Do you mean visually diffing between two numbers? I can see how that would make sense yeah! Maybe we can word it better...

Reply:
Yeah, exactly.  Ie, it's easier to see the true size of the values. Feel free to remove that part though if it's too vague.

<img width="138" alt="Friday 02 June" src="https://github.com/raunofreiberg/interfaces/assets/10933880/840963d3-9dcc-4211-ac8d-03d30b6dc829">

Reply:
I like this but would mention `font-variant-numeric` :)